validation/src/main/resources/application.conf

```diff
@@ -1,6 +1,7 @@
 akka {
-  stdout-loglevel = "DEBUG"
-  loglevel = "OFF"
+  stdout-loglevel = "OFF"
```

Comment:
Not exactly. This is silenced only when the logs are being sent to a file. When logging to a file, these startup logs show up at the console always. In any case, I can see the startup logs just fine